.github/workflows/ci.yaml

```diff
@@ -10,11 +10,11 @@ name: CI
 # fork in github, minimizing noise for maintainers. This
 # workflow also runs on nightly basis at 12:00 AM (00:00 UTC)
 
-on:  # yamllint disable-line rule:truthy
+on: # yamllint disable-line rule:truthy
   push:
   pull_request:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: "0 0 * * *"
```

Comment:
This and the above change most likely came in due to your editor settings but they are failing the linter. Can you please revert these lines?